### ISSUE TITLE
No sigmoid on logits?

### ISSUE THREAD

**Issue by shuki-hpcnt** [dr-costas/dnd-sed#1]
It seems like that metric functions are using direct logit values from `Linear` and calculate positive/negative by thresholding with 0.5, but I cannot find any `sigmoid` or similar variants that transforms logits into probability estimations. Am I missing something?

**Comment by dr-costas** [comment]
Hi, 

No, you are absolutely right, this is a bug. The thing is that this is not the only one. For example, there is no `f1_func` argument at the call of `training` function, file `experiments/_processes.py`, line 287. 

The bugs are caused because the repository is a refactored version of the code used in the experiments, mainly cleaned and structured in a more friendly way for other people. The purpose of the initial form of repo was to showcase to the reviewers the code and prove that the code will be released publicly. 

I'm planning to fix the known bugs by this Monday. 

I'm sorry for that. I will let you know (by posting in this issue) when I push the fixes.

**Comment by shuki-hpcnt** [comment]
Thanks for the reply! Looking forward to the fixes. BTW, the other bug that you mentioned is not the bug since there is `**common_kwargs` which includes `f1_func` argument. I actually cloned the code and succeeded to run the experiment :D

**Comment by dr-costas** [comment]
> BTW, the other bug that you mentioned is not the bug since there is **common_kwargs which includes f1_func argument. I actually cloned the code and succeeded to run the experiment :D

I must have fixed it already, good to know! :D

**Comment by dr-costas** [comment]
Fixed. Thank you for spotting that. 

I will close this issue, please feel free to make a new issue if you spot something else.
